modules/mongodb/mongodb.go

```diff
@@ -0,0 +1,114 @@
+package mongo
+
+import (
+	"context"
+	"time"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func init() {
+	module.Register("mongodb", module.Creator{
+		Defaults: module.Defaults{
+			UpdateEvery:        module.UpdateEvery,
+			AutoDetectionRetry: module.AutoDetectionRetry,
+			Priority:           module.Priority,
+			Disabled:           false,
+		},
+		Create: func() module.Module { return New() },
+	})
+}
+
+func New() *Mongo {
+	return &Mongo{
+		Config: Config{
+			Timeout: defaultTimeout,
+			Uri:     defaultUri,
```

Comment:
My suggestion is to remove both constants and use string values instead. In that case, you don't need to test these config options in TestNew. Whatever is set in New is valid.

Reply:
I am not sure about this. Ping will actual send a ping command to the database and see if it's live or not. We do that anyway in the `Check` method where we call `Collect` and this will fail if the database is don't or configuration is wrong. I was actually thinking do a ping in the check method instead of collect because it's more lightweight. What do you think?

Reply:
done